src/test/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/impl/RegexBasedEdgeLabelMappingToURIsImplTest.java

```diff
@@ -0,0 +1,48 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class RegexBasedEdgeLabelMappingToURIsImplTest {
+    protected final String NSRELATIONSHIP = "https://example2.org/test/";
+    protected final String regex = "^\\w+";
+
+    protected final EdgeLabelMapping edgeLabelMapping = new RegexBasedEdgeLabelMappingToURIsImpl(regex, NSRELATIONSHIP);
+
+    @Test
+    public void mapEdgeLabel() {
```

Comment:
Can you please also include a test in which the given edge label does not match the regular expression and, thus, an exception is thrown.